core/src/main/resources/hudson/PluginManager/installed_pl.properties

```diff
@@ -30,13 +32,22 @@ Version=Wersja
 downgradeTo=Powr\u00F3\u0107 do wersji {0}
 requires.restart=Wymagane jest ponowne uruchomienie Jenkinsa. Zmiany wtyczek w tym momencie s\u0105 bardzo niewskazane. Uruchom ponownie Jenkinsa, zanim wprowadzisz zmiany.
 Uninstallation\ pending=Trwa odinstalowywanie
+detached-disable=Wy\u0142\u0105czenie tej wtyczki mo\u017Ce nie by\u0107 bezpieczne
+This\ plugin\ cannot\ be\ enabled=Ta wtyczka nie mo\u017Ce by\u0107 w\u0142\u0105czona
 This\ plugin\ cannot\ be\ disabled=Ta wtyczka nie mo\u017Ce by\u0107 wy\u0142\u0105czona
+It\ has\ one\ or\ more\ enabled\ dependents=Ma jedn\u0105 lub wiele zainstalowanych zale\u017Cno\u015Bci
+This\ plugin\ cannot\ be\ uninstalled=Ta wtyczka nie mo\u017Ce by\u0107 odinstalowana
+It\ has\ one\ or\ more\ installed\ dependents=Ma jedn\u0105 lub wiele zainstalowanych zale\u017Cno\u015Bci
 No\ plugins\ installed.=Brak zainstalowanych wtyczek
-This\ plugin\ cannot\ be\ enabled=Ta wtyczka nie mo\u017Ce by\u0107 w\u0142\u0105czona
```

Comment:
Moved above to keep together with other translations which reflect to same scope